### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DimaFlacko**

### CODE REVIEW

File: src/sortStudents.ts [hunk 1]
```diff
@@ -1,16 +1,58 @@
...
+          : Number(curr[sortBy]) - Number(prev[sortBy]);
+
+      case SortType.AverageGrade:
+        return order === 'asc'
+          ? prev[sortBy]
+            .reduce((a, b) => a + b) / prev[sortBy].length - curr[sortBy]
+            .reduce((a, b) => a + b) / curr[sortBy].length
+          : curr[sortBy]
+            .reduce((a, b) => a + b) / curr[sortBy].length - prev[sortBy]
+            .reduce((a, b) => a + b) / prev[sortBy].length;
```
Comment (Andrew-Skoryk) [inline src/sortStudents.ts:50]: Would be better to use this logic in a separate function

**Review by Andrew-Skoryk — APPROVED** [review]
GJ!

File: src/sortStudents.ts [hunk 2]
```diff
@@ -1,16 +1,58 @@
...
+      case SortType.Name:
+      case SortType.Surname:
+        return prev[sortBy].localeCompare(curr[sortBy]);
+
+      case SortType.Age:
+        return order === 'asc'
+          ? prev[sortBy] - curr[sortBy]
+          : curr[sortBy] - prev[sortBy];
+
+      case SortType.Married:
```
Comment (Denchepornyuk) [inline src/sortStudents.ts:38]: you can combine these cases into one

File: src/sortStudents.ts (hunk 1 above)
Comment (Denchepornyuk) [reply src/sortStudents.ts:50]: Agree, let's create a function to repeat code

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**